Main.java

```diff
@@ -9,7 +11,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create the scanner for user input 
+    Scanner input = new Scanner(System.in);
+    
+    System.out.println("please enter first x coordinate"); 
+    int X1 = input.nextInt();
+    System.out.println("please enter first y coordinate");
+    int Y1 = input.nextInt();
+    System.out.println("please enter second x coordinate");
+    int X2 = input.nextInt();
+    System.out.println("please enter second y coordinate");
+    int Y2 = input.nextInt();
+
+    final double total = (Y2 - Y1) / (X2 - X1);     
```

Comment:
Don't forget to add comments to your code! Also remember that variable names always begin with a lowercase letter. Finally, can you have a point where the x or y coordinate contains a decimal place? These should probably be doubles instead of ints. We know this because the sample output shows a slope of 2.0 instead of just 2.